packages/twenty-front/src/hooks/useRtl.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react';
+import { useRecoilValue } from 'recoil';
+import { isRtlLocale } from 'twenty-shared/utils';
+
+import { dateLocaleState } from '~/localization/states/dateLocaleState';
+
+/**
+ * Hook that synchronises the document direction with the current locale.
+ *
+ * When the locale is RTL the document's `dir` attribute is set to `rtl`,
+ * otherwise it defaults to `ltr`.
+ */
+export const useRtl = () => {
```

Comment:
bad pattern, we should not use an effect to do that. Instead when we get the locale from the user, we should synchornously set it in document.